docs/migrating.rst

```diff
@@ -1,35 +1,306 @@
 .. currentmodule:: discord
 
-.. _migrating_1_0:
+.. _migrating_2_0:
```

Comment:
General thingy maybe rename it to migrating_to_v2 to make making another one easier in the future

Reply:
migrating_to_2_0 includes the minor version specifier too, which is nice. Also, it's the syntax used previously so it makes sense to keep it.